src/components/TextArea/TextArea.jsx

```diff
@@ -0,0 +1,60 @@
+import PropTypes from "prop-types";
+import React from "react";
+import { classNames } from "../../utils";
+import Icon from "../Icon/Icon.jsx";
+import styles from "./styles.css";
+
+const TextArea = ({
+  type,
+  label,
+  id,
+  error,
+  wrapperClassNames,
+  inputClassNames,
+  ...props
+}) => {
+  return (
+    <div className={classNames(styles.textAreaWrapper, wrapperClassNames)}>
+      <textarea
+        id={id}
+        placeholder=" "
+        className={classNames(
+          styles.textArea,
+          error && styles.textAreaError,
+          inputClassNames
+        )}
+        type={type}
+        {...props}
+      />
+      <Icon
+        type="alert"
+        backgroundColor="#ed6d47"
```

Comment:
We are these colors coming from ? and how does they work on dark theme ?

Reply:
They were taken from the TextInput component, and they work just like the text input icons on dark theme:
https://github.com/decred/pi-ui/blob/1910f0892dc69fd6380afc7bb0b6882e9c269d91/src/components/TextInput/TextInput.jsx#L34-L35